tools/container/runtime/containerd/containerd.go

```diff
@@ -109,11 +112,13 @@ func Setup(c *cli.Context, o *container.Options, co *Options) error {
 }
 
 // Cleanup reverts a containerd configuration to remove the nvidia-containerd-runtime and reloads it
-func Cleanup(c *cli.Context, o *container.Options, co *Options) error {
+func Cleanup(c *cli.Context, hostRoot string, o *container.Options, co *Options) error {
 	log.Infof("Starting 'cleanup' for %v", c.App.Name)
 
+	containerdCommand := getContainerdConfigCommand(hostRoot)
 	cfg, err := containerd.New(
 		containerd.WithPath(o.Config),
+		containerd.WithConfigSource(toml.FromCommandLine(containerdCommand)),
```

Comment:
It would be great to be able to write the following instead:
```suggestion
		containerd.WithConfigSource(containerd.CommandLineSource()),
```

Reply:
Also, I understand that we're using the existing cleanup logic, but we can also think about how we want to improve this in a follow up now that we have the previous config available to properly store somewhere. May be more tricky with different sources though.

Reply:
I've made this change